dogfood/main.tf

```
@@ -11,6 +11,11 @@ terraform {
   }
 }
 
+variable "datocms_api_token" {
+  type = string
+  description = "An API token from DATOCMS for usage with building our website."
```

```suggestion
  description = "An API token from DATOCMS for usage with building our website."
  default = ""
```

This removes prompts on `coder template push`